### DISCUSSION TITLE
I propose to change the design a little - the function is not visible

### DISCUSSION THREAD

**Discussion by MikleBas** [magicbug/Cloudlog#2021] [Ideas / Feature Requests]
Where it is now - it is not visible at all during work
I propose to move the block as in the picture
I think it will make more sense

![Screenshot (1)](https://user-images.githubusercontent.com/99086477/222705771-56bdc2d8-5e42-4629-bffe-e0c25fdab42d.png)

![Screenshot](https://user-images.githubusercontent.com/99086477/222707260-4b8118e4-c7a9-4bf2-b8ca-2de4e8a56e6f.png)

**Comment by R1BLH** [comment]
Just click to the "DXCC Summary for..."

![изображение](https://user-images.githubusercontent.com/16667173/222760662-57a2a4db-8039-4a02-904a-69e50c957808.png)

**Comment by MikleBas** [comment]
Yes, not obvious
I didn't even notice it was a link :)
But still - I can't see this field when I'm working...
Rather, you can see if you scroll the page, but as a rule I don’t do this directly to the radio connection, this has nothing to do with it and I don’t waste attention on it

**Reply by R1BLH** [reply]
I discovered this for myself unexpectedly. The cursor does not change when you move the mouse over this field.
In any case, the field on the right shows data about the previous QSOs with this callsign. And somewhere else a photo from qrz.com should be displayed, but I could not get it to work.

**Reply by phl0** [reply]
Submitted a patch to show link by different mouse pointer type here: https://github.com/magicbug/Cloudlog/pull/2023

**Reply by phl0** [reply]
qrz.com image stuff may only work if you have a premium/xml subscription to qrz.com ...

**Reply by R1BLH** [reply]
> qrz.com image stuff may only work if you have a premium/xml subscription to qrz.com ...

I see. I was just curious to try it out. On the other hand, it is not an essential feature, just decoration.

**Reply by MikleBas** [reply]
> Submitted a patch to show link by different mouse pointer type here: #2023

I hope the edits will automatically catch up with the next update of the scripts?
